app/src/main/java/nu/yona/app/enums/NotificationMessageEnum.java

```diff
@@ -20,7 +20,7 @@
 	/**
 	 * Buddy connect request accepted notification message enum.
 	 */
-	BUDDY_CONNECT_REQUEST_ACCEPTED(NotificationEnum.BUDDYCONNECTREQUESTMESSAGE, StatusEnum.ACCEPTED, getString(R.string.buddyconnectrequested), R.drawable.icn_ok),
+	BUDDY_CONNECT_REQUEST_ACCEPTED(NotificationEnum.BUDDYCONNECTREQUESTMESSAGE, StatusEnum.ACCEPTED, getString(R.string.buddyresponseaccepted), R.drawable.icn_ok),
```

Comment:
These resource names are very confusing. Let's look at what pairs exist:
* There are connect requests and connect response messages.
* The status can be accepted or rejected
* There are messages sent by you and sent to you

The flow is like this:
* Richard sends a connect request message to Bob. Richard now has no messages, Bob has one. The status in that message is "requested".
* Bob now can accept or reject that message. This will cause the status of that message to become accepted or rejected and Richard will receive a buddy connect response message with the status accepted or rejected.

Given this, I propose to define the following enum values resources:
* ``BUDDY_CONNECT_REQUEST_OPEN``, ``buddyrequestopen``
* ``BUDDY_CONNECT_REQUEST_ACCEPTED``, ``buddyrequestaccepted``
* ``BUDDY_CONNECT_REQUEST_REJECTED``, ``buddyrequestrejected``
* ``BUDDY_CONNECT_RESPONSE_ACCEPTED``, ``buddyresponseaccepted``
* ``BUDDY_CONNECT_RESPONSE_REJECTED``, ``buddyresponserejected``

Makes sense?